### DISCUSSION TITLE
still working ?!

### DISCUSSION THREAD

**Discussion by Mobin-khorushi** [def-studio/telegraph#434] [Q&A]
is this still working ?! i followed instructions as mentioned ! even created webhook perfectly so 

https://api.telegram.org/my_token/getWebhookInfo

returns data perfectly expect one thing ! 

"last_error_message": "Wrong response from the webhook: 404 Not Found",

**Comment by fabio-ivona** [comment]
Hi, 404 error is returned from webhook when it can retrieve your bot
